audio/sdl2-audio/source/main.c

```diff
@@ -64,6 +64,8 @@ int main(int argc, char *argv[])
     // Shuts down SDL subsystems
     SDL_Quit();
 
+    romfsExit();
```

Comment:
Slightly nitpicky but can you move this down below the Deinitialize comment - it is libnx deinitialization code after all.